drivers/sensor/ens210/ens210.c

```diff
@@ -196,12 +235,12 @@ static int ens210_init(struct device *dev)
 {
 	struct ens210_data *drv_data = dev->driver_data;
 	const struct ens210_sens_run sense_run = {
-		.t_run = 1,
-		.h_run = 1
+		.t_run = ENS210_T_RUN,
+		.h_run = ENS210_H_RUN
 	};
 	const struct ens210_sens_start sense_start = {
-		.t_start = 1,
-		.h_start = 1
+		.t_start = ENS210_T_RUN,
```

Comment:
ENS210_T_START

Reply:
I know this looks strange, but if I have temperature in single-shot mode, then `ENS210_T_START` is set to `1` and a single measurement is performed by the end of the initialization. However, setting it to `ENS210_T_RUN` only starts the continuous measurements, but not the single-shot measurements.